tests/BasicsTest.php

```diff
@@ -0,0 +1,31 @@
+<?hh
+
+class BasicsTest extends PHPUnit_Framework_TestCase {
+  public function testDivWithString() {
+    $xhp =
+      <div>
+        Hello, world.
+      </div>;
+    $this->assertEquals('<div> Hello, world. </div>', $xhp->toString());
+  }
+
+  public function testDivWithChild() {
+    $xhp = <div><div>Herp</div></div>;
+    $this->assertEquals('<div><div>Herp</div></div>', $xhp->toString());
+  }
+
+  public function testInterpolation() {
```

Comment:
Perhaps a case along these lines that verifies that HTML in strings is properly escaped?